docs/getting-started/security-ui.asciidoc

```diff
@@ -99,6 +99,19 @@ image::images/network-ui.png[Shows the Network page]
 [role="screenshot"]
 image::images/users/users-page.png[Shows the Users page]
 
+[float]
+=== Intelligence
+
+The Intelligence section contains the Indicators page, which collects data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<investigate-events>> to learn more.
```

Comment:
```suggestion
The Intelligence section contains the Indicators page, which displays data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<investigate-events>> to learn more.
```
Collect seems to me like something that server agents do

Reply:
@benironside I think it's trying convey that the page gathers data from disparate sources/feeds and presents them in a single place. "Displays" sounds like the data might have already been there, it just needed to be displayed.

@nastasha-solomon Any preference? This description was lifted from [your PR](https://github.com/elastic/security-docs/pull/2526) on the Indicators page.

Reply:
Hm, I see your point @benironside but think I'd still prefer to keep "collects". The second part of the sentence already tells users that the page displays data ingested from enabled threat feeds.